messaging/service-worker.js

```diff
@@ -10,8 +10,10 @@ function initInSw() {
   // Give the service worker access to Firebase Messaging.
   // Note that you can only use Firebase Messaging here. Other Firebase libraries
   // are not available in the service worker.
-  importScripts('https://www.gstatic.com/firebasejs/9.2.0/firebase-app.js');
-  importScripts('https://www.gstatic.com/firebasejs/9.2.0/firebase-messaging.js');
+  importScripts('https://www.gstatic.com/firebasejs/8.10.1/firebase-app.js');
+  importScripts(
+    'https://www.gstatic.com/firebasejs/8.10.1/firebase-messaging.js'
+  );
```

Comment:
Can we keep the original formatting?
```suggestion
  importScripts('https://www.gstatic.com/firebasejs/8.10.1/firebase-messaging.js');
```